tools/donate-cpu.py

```diff
@@ -159,11 +160,14 @@ def wget(url, destfile, bandwidth_limit):
         else:
             print('Error: ' + destfile + ' exists but it is not a file! Please check the path and delete it manually.')
             sys.exit(1)
-    limit_rate_option = ''
+    wget_call = ['wget', '--verbose', '--tries=10', '--timeout=300', '-O', destfile, url]
```

Comment:
`--verbose` is normally already the default. So it is not really necessary to add it explicitly. Otherwise it does not harm.

Reply:
A debugging left-over